var/spack/repos/builtin/packages/flux-core/package.py

```diff
@@ -91,7 +91,8 @@ class FluxCore(AutotoolsPackage):
     # `link` dependency on python due to Flux's `pymod` module
     depends_on("python@3.6:", type=("build", "link", "run"))
     # Use of distutils in configure script dropped in v0.55
-    depends_on("python@:3.11", when="@:0.54", type=("build", "link", "run"))
+    # Issues with detection of cffi version, may have been fixed in newer versions?
+    depends_on("python@:3.11", type=("build", "link", "run"))
```

Comment:
@trws @Chrismarsh this is my lazy "fix" for flux-core. I don't really like patching this when I don't know what's going on, and it seems like this may have been fixed at some point. If anyone would like to open a separate PR to get flux-core building with Python 3.12, that would be great, but I don't think it should hold up py-cffi supporting Python 3.12.

Reply:
I patched it upstream in version 0.67.0, any chance you'd be willing to take this tweak?

```suggestion
    depends_on("python@:3.11", when="@:0.67", type=("build", "link", "run"))
```